README.md

```diff
@@ -45,6 +45,7 @@ You can set different configuration options to control the format of GUIDs that
 * insertGuid.showLowercase: Show lowercase GUIDs (with and without braces) when presenting possible GUID formats to insert. The default is `true`.
 * insertGuid.showUppercase: Show uppercase GUIDs (with and without braces) when presenting possible GUID formats to insert. The default is `false`.
 * insertGuid.showCodeSnippets: Show code snippets for C++ when presenting possible GUID formats to insert. The default is `true`.
+* insertGuid.includeHyphens: Include hyphens in GUID formats. The default is `true`.
```

Comment:
Why is this an option over just including the one extra format? The previous options would've added several permutations, but since I've never seen a `{guid}` without hyphens, seems this is only one extra option for just `guid` without hyphens.

Using an option means you have to change it if you want a different format, which is tedious.

Reply:
I guess I was thinking to avoid adding a bunch of permutations, but I think you're correct.
Thoughts on ordering of the basic formats?